sdk/monitor/azquery/utils_test.go

```diff
@@ -101,20 +80,10 @@ func run(m *testing.M) int {
 		}
 
 	}
-	if recording.GetRecordMode() == recording.RecordingMode {
-		err := recording.AddGeneralRegexSanitizer(fakeWorkspaceID, workspaceID, nil)
-		if err != nil {
-			panic(err)
-		}
-		err = recording.AddGeneralRegexSanitizer(fakeWorkspaceID2, workspaceID2, nil)
-		if err != nil {
-			panic(err)
-		}
-		err = recording.AddGeneralRegexSanitizer(fakeResourceURI, resourceURI, nil)
-		if err != nil {
-			panic(err)
-		}
-	}
+	workspaceID = getEnvVar("WORKSPACE_ID", fakeWorkspaceID)
+	workspaceID2 = getEnvVar("WORKSPACE_ID2", fakeWorkspaceID2)
+	resourceURI = getEnvVar("WORKSPACE_ID", fakeResourceURI)
```

Comment:
```suggestion
	resourceURI = getEnvVar("RESOURCE_URI", fakeResourceURI)
```

Reply:
Good catch, thanks! Fixed in [d0c65c0](https://github.com/Azure/azure-sdk-for-go/pull/21965/commits/d0c65c005478be252e03a390bdc8d4ded5ffe04a)